browser/css/jsdialogs.css

```diff
@@ -1510,10 +1510,22 @@ input[type='number']:hover::-webkit-outer-spin-button {
 	margin: auto;
 }
 
+[data-theme='dark'] .ui-image {
+	filter: invert(.95)
+}
+
 .ui-drawing-area-container {
 	margin: auto;
 }
 
+[data-theme='dark'] .ui-drawing-area-container {
+	filter: invert(.9)
+}
+
+[data-theme='dark'] img.notebookbar {
+	filter: invert(.9)
+}
+
```

Comment:
This or any CSS color graphical effects cannot enter if they affect element that:
1. Can have any other color than black or white (otherwise it will be impossible to see/read)
2. Even if some elements can only be monochrome they should stay faithful to how the styles appear when printed (standard used also in MSO etc)